src/openforms/forms/models/form_variable.py

```diff
@@ -0,0 +1,107 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from ..constants import (
+    FormVariablesDataTypes,
+    FormVariablesInitialValues,
+    FormVariablesSources,
+)
+from .form import Form
+from .form_definition import FormDefinition
+
+
+def get_today():
+    return timezone.now().isoformat()
+
+
+STATIC_INITIAL_VALUES = {FormVariablesInitialValues.today: get_today}
+
+
+class FormVariable(models.Model):
+    form = models.ForeignKey(
+        to=Form,
+        verbose_name=_("form"),
+        help_text=_("Form to which this variable is related"),
+        on_delete=models.CASCADE,
+    )
+    form_definition = models.ForeignKey(
+        to=FormDefinition,
+        verbose_name=_("form definition"),
+        help_text=_(
+            "Form definition to which this variable is related. This is kept as metadata"
+        ),
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    name = models.CharField(
+        verbose_name=_("name"),
+        help_text=_("Name of the variable"),
+        max_length=100,
+    )
+    # TODO Ideally should be namespaced with the form slug. Should add validator for format <form_slug>:<key>?
+    # and the key part should be of slug format
```

Comment:
I think I changed my mind about this while looking at the front end. I will add a unique_together constraint with for key/form and add the validation to the list serializer (https://github.com/encode/django-rest-framework/issues/6395)